src/coreclr/jit/morph.cpp

```diff
@@ -1865,26 +1919,16 @@ GenTree* Compiler::fgInsertCommaFormTemp(GenTree** ppTree, CORINFO_CLASS_HANDLE
 {
     GenTree* subTree = *ppTree;
 
-    unsigned lclNum = lvaGrabTemp(true DEBUGARG("fgInsertCommaFormTemp is creating a new local variable"));
-
-    if (varTypeIsStruct(subTree))
-    {
-        assert(structType != nullptr);
-        lvaSetStruct(lclNum, structType, false);
-    }
-
     // If subTree->TypeGet() == TYP_STRUCT, gtNewTempAssign() will create a GT_COPYBLK tree.
     // The type of GT_COPYBLK is TYP_VOID.  Therefore, we should use subTree->TypeGet() for
     // setting type of lcl vars created.
-    GenTree* asg = gtNewTempAssign(lclNum, subTree);
-
-    GenTree* load = new (this, GT_LCL_VAR) GenTreeLclVar(GT_LCL_VAR, subTree->TypeGet(), lclNum);
+    TempInfo tempInfo = fgMakeTemp(subTree, structType);
```

Comment:
Comment two lines up is now stale, should refer to `fgMakeTemp`.

Reply:
Fixed. I moved the comment to inside of `fgMakeTemp` and modified it a tiny bit.